modules/surface_matching/src/c_utils.hpp

```diff
@@ -85,9 +85,13 @@ static inline void TCross(const double a[], const double b[], double c[])
 
 static inline double TAngle3(const double a[3], const double b[3])
```

Comment:
I think the name of this function should be changed accordingly. Something like `TAngle3Normalized` is more suitable.